### PR TITLE
test(editor): Add e2e test to verify Save button behavior when opening and closing Webhook node

### PR DESCRIPTION
**Pull request by CharlieKolb**
## Summary

Document canvas v1 behavior that did not resurface in canvas v2.

## Related Linear tickets, Github issues, and Community forum posts

https://linear.app/n8n/issue/ADO-2270/bug-opening-webhook-ndv-marks-workflow-as-unsaved

## Review / Merge checklist

- [x] PR title and summary are descriptive. ([conventions](../blob/master/.github/pull_request_title_conventions.md)) 
- [ ] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [x] Tests included. 
- [ ] PR Labeled with `release/backport` (if the PR is an urgent fix that needs to be backported)

### CODE REVIEW

File: cypress/e2e/2270-ADO-opening-webhook-ndv-marks-workflow-as-unsaved.cy.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+const ndv = new NDV();
+
+describe('ADO-2270 Save button resets on webhook node open', () => {
+	it('should not reset the save button if webhook node is opened and closed', () => {
+		workflowPage.actions.visit();
+		workflowPage.actions.addInitialNodeToCanvas('Webhook');
+		workflowPage.getters.saveButton().click();
+		workflowPage.actions.openNode('Webhook');
+
+		ndv.actions.close();
```
Comment (mutdmour) [inline cypress/e2e/2270-ADO-opening-webhook-ndv-marks-workflow-as-unsaved.cy.ts:13]: would be nice if you can use `cypress/composables/ndv` pattern here
`closeNDV()` instead for  example.
same with workflow object. but just a suggestion. 

We are trying to move away form these classes

**Review by mutdmour — DISMISSED** [review]

File: cypress/e2e/2270-ADO-opening-webhook-ndv-marks-workflow-as-unsaved.cy.ts (hunk 1 above)
Comment (CharlieKolb) [reply cypress/e2e/2270-ADO-opening-webhook-ndv-marks-workflow-as-unsaved.cy.ts:13]: Good to know, thanks! Will push a change

Comment (CharlieKolb) [reply cypress/e2e/2270-ADO-opening-webhook-ndv-marks-workflow-as-unsaved.cy.ts:14]: Ended up going with the original version as equivalent functions with `composables` are currently broken on canvas v2.

**Review by mutdmour — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@1.66.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.66.0)

**Pull request merged**